microhapulator/tests/data/def/off-target-offsets.tsv

```diff
@@ -0,0 +1,148 @@
+Marker	Offset	Chrom	GenomeOffset
```

Comment:
Maybe `GRCh38Offset` or `OffsetHg38` or something like that instead of `GenomeOffset`.

Reply:
Changed the column heading to OffsetHg38